### PR TITLE
Update 08.responserules.md - clarify response rule group supported

### PR DESCRIPTION
**Pull request by gtam**
Address groups like criteria of address=x.x.x.x/x is not supported for matching.

### CODE REVIEW

File: docs/05.policy/08.responserules/08.responserules.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Response Rules provide a flexible, customizable rule engine to automate response
 
 Creating a new Response Rule using the following:
 
-+ Group. A rule will apply to a Group. Please see the section Run-Time Security Policy -> Groups for more details on Groups and how to create a new one if needed.
++ Group. A rule will apply to a container Group (address group is not supported, for example criteria address=x.x.x.x/x). Please see the section Run-Time Security Policy -> Groups for more details on Groups and how to create a new one if needed.
```
Comment (sunilarjun) [inline docs/05.policy/08.responserules/08.responserules.md:16]:
```suggestion
+ Group. A rule will apply to a container Group (address group is not supported, for example, criteria address=x.x.x.x/x). Please see the section [Run-Time Security Policy -> Groups](../04.groups/04.groups.md) for more details on Groups and how to create a new one if needed.
```

**Review by sunilarjun — CHANGES_REQUESTED** [review]
Hello! Just some minor changes and added a relative link to the Groups page, let me know if it looks alright, thank you.

File: docs/05.policy/08.responserules/08.responserules.md (hunk 1 above)
Comment (gtam) [reply docs/05.policy/08.responserules/08.responserules.md:16]: 👍

Comment (sunilarjun) [reply docs/05.policy/08.responserules/08.responserules.md:16]: Hi @gtam , are you able to make the update to the content? I can merge in once updated, thank you!

**Review by sunilarjun — APPROVED** [review]
Thank you for addressing the changes!

**Pull request merged**